pymc/distributions/continuous.py

```diff
@@ -3018,9 +3027,9 @@ class VonMises(CircularContinuous):
 
     Parameters
     ----------
-    mu: float
+    mu : tensor_like of float, default 0
         Mean.
-    kappa: float
+    kappa : tensor_like of float, optional
```

Comment:
same comment here about kappa not being really optional. I think the None in the call signature (also in the other functions) is used so that both options below work:

```
VonMises("name", 2, 1)
VonMises("name", kappa=3)
```
but as you see, kappa is not really optional and needs to be given a value.